core/common/src/test/java/alluxio/ConfigurationRule.java

```diff
@@ -48,6 +48,29 @@ public ConfigurationRule(final PropertyKey key, final Object value, InstancedCon
     }, conf);
   }
 
+  /**
+   * Set a specific PropertyKey of the global configuration,
+   * change is only visible in the scope of the calling method.
+   *
+   * @param key the key of the configuration property to set
+   * @param value the value to set it to, can be null to unset
+   */
+  public void set(final PropertyKey key, final Object value) {
```

Comment:
nit: can return `this` so as to enable method chaining.

Reply:
Fixed